docs/roadmap.md

```diff
@@ -2,23 +2,27 @@
 
-### App Details Page UI Update
+### Custom Workload Profile features
 
-Following several months of user testing and customer feedback we are planning on introducing a number of changes to improve the usability, predictability and scalability of our UI. The focus lies on the App Details Page, the control centre of an App. We want to address issues related to interaction design, information hierarchy, status communication as well as aesthetics while touching on UI patterns which are likely to change the look and feel of the Humanitec UI as a whole.
+[Workload Profiles](https://docs.humanitec.com/reference/workload-profiles) are a way to define the baseline configuration of a workload. Humanitec provides a number of default workload profiles and users have been able to add their own. Up to this point, platform teams have been limited to the UI provided by Humanitec. Custom Workload Profile features will allow platform teams to define schemas for their features and annotate these schemas to generate custom UI elements. This will allow platform teams to fully customize the developer experience for their Custom Workload Profiles.
 
-### Increase Resource Driver Range And Flexibility
+### Cost Control
 
-Resource Drivers are used to provision things that are consumed by workloads. Most commonly this is infrastructure such as databases or dns. There are a growing number of drivers available out-of-the-box in Humanitec. In this quarter we aim to improve the flexibility and range of our driver offering. This includes adding generic drivers that work with templates and IaC as well as open-sourcing many of our drivers. We also plan to extend how drivers can be defined - allowing drivers to be defined in terms of other drivers. This should make it easier for ops teams to define good defaults to reuse within their teams. A comprehensive overview of the Humanitec driver ecosystem as well as its integration and contribution options will be published in a central hub on our website.
+Humanitec enables self-service for engineering teams by allowing developers to spin up new environments and provision resources like databases, file storage, clusters, and many more on the fly. However, not all of those resources are used all the time, and enabling teams to oversee and control their infrastructure usage better can generate significant savings. The new Cost Control capabilities such as on-demand and scheduled environment pausing will help to reduce spendings on idle resources while maintaining the same level of self-service for developers.
```

Comment:
This scope is way to broad. The only thing that our initial cost control with cover is Compute.

I would re-write this as:
> Humanitec allows developers to spin up entire environments on demand. However, not all of those environments are used all the time and idle environments still cost money. The new Cost Control capabilities such as on-demand and scheduled environment pausing will help to reduce spend on idle environments. At the same time, developers can still spin up paused environments on demand so they are not blocked.

Reply:
Thanks!